ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/RetryMechanism.java

```diff
@@ -15,11 +15,19 @@
 import io.vertx.core.buffer.Buffer;
 import io.vertx.core.http.HttpClientResponse;
 
-public interface RetryMechanism<T> {
+public abstract class RetryMechanism<T> {
```

Comment:
Is there any need to use generics on the RetryMechanism?

Reply:
nope!